easybuild/easyconfigs/t/TensorFlow-Graphics/TensorFlow-Graphics-2021.12.3-foss-2021b-CUDA-11.4.1.eb

```diff
@@ -0,0 +1,62 @@
+easyblock = 'PythonBundle'
+
+name = 'TensorFlow-Graphics'
+version = '2021.12.3'
+versionsuffix = '-CUDA-%(cudaver)s'
+
+homepage = 'https://github.com/tensorflow/graphics'
+description = """Tensorflow Graphics provides a set of differentiable graphics
+and geometry layers (e.g. cameras, reflectance models, spatial transformations,
+mesh convolutions) and 3D viewer functionalities (e.g. 3D TensorBoard) that can
+be used to train and debug your machine learning models of choice."""
+
+toolchain = {'name': 'foss', 'version': '2021b'}
+toolchainopts = {'pic': True}
+
+dependencies = [
+    ('CUDA', '11.4.1', '', SYSTEM),
+    ('Python', '3.9.6'),
+    ('TensorFlow', '2.7.1', versionsuffix),
+    ('TensorFlow-Datasets', '4.8.3', versionsuffix),
+    ('matplotlib', '3.4.3'),
+    ('h5py', '3.6.0'),
+    ('tqdm', '4.62.3'),
+    ('OpenEXR', '3.1.1'),
+    ('networkx', '2.6.3'),
+]
+
+use_pip = True
+sanity_pip_check = True
+
+exts_list = [
+    ('typeguard', '2.13.3', {
+        'checksums': ['00edaa8da3a133674796cf5ea87d9f4b4c367d77476e185e80251cc13dfbb8c4'],
+    }),
+    ('tensorflow-addons', '0.19.0', {
+        'sources': [{
+            'source_urls': ['https://github.com/tensorflow/addons/archive/refs/tags/'],
+            'download_filename': 'v%(version)s.tar.gz',
+            'filename': '%(name)s-%(version)s.tar.gz',
+        }],
+        'checksums': ['6e1c40f03c9a35453a26eacbf36700f9666882418058a2f0d81aaa939135262f'],
+    }),
+    ('OpenEXR', '1.3.9', {
+        'modulename': 'Imath',
+        'patches': ['TensorFlow-Graphics-2021.12.3_openexr-setup.patch'],
+        'preinstallopts': "env | grep '^EB' && cat setup.py && ",
```

Comment:
```suggestion
```

Remove unneeded `preinstallopts`. I guess these were use to help create the patch?